client/src/pages/ErrorPage/ErrorPage.js

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import { Box, Typography } from '@mui/material';
+import Button from '@mui/material/Button';
+import { useNavigate } from 'react-router-dom';
+import styled from '@emotion/styled';
 
-const ErrorPage = () => <div>Error</div>;
+const ErrorContainer = styled('dix')({
+  width: '100%',
+  height: '100%',
+  textAlign: 'center',
+  '& > div': {
+    display: 'flex',
+    justifyContent: 'center',
+    '& > img': {
+      width: '50vw',
+      maxWidth: '500px',
+    },
+  },
+  '& > h2': {
+    fontSize: '25px',
+    lineHeight: '30px',
+    marginBottom: '30px',
+  },
+  '& > p': {
+    fontSize: '15px',
```

Comment:
the same as above. not needed.  the default style is as below:

![image](https://user-images.githubusercontent.com/107735322/199251447-396a276c-1507-41b1-a583-45d61723a2e8.png)